global.json

```
@@ -5,10 +5,7 @@
   "tools": {
     "dotnet": "5.0.100-alpha1-013788",
     "runtimes": {
-      "dotnet/x64": [
-        "$(MicrosoftNETCoreAppRuntimeVersion)"
-      ],
-      "dotnet/x86": [
+      "dotnet": [
```

```suggestion
      "dotnet/x64": [
        "$(MicrosoftNETCoreAppRuntimeVersion)"
      ],
      "dotnet/x86": [
        "$(MicrosoftNETCoreAppRuntimeVersion)"
      ],
      "dotnet/arm64": [
```

Parts of the full build need to cross-compile and test on alternative platforms. So, let's instead download it all.

---

I was afraid of this. This is the first patch I tried. Unfortunately, that doesn't work :(

```
Setting msbuild verbosity to minimal                                                                                  
Downloading 'https://dot.net/v1/dotnet-install.sh'                                                                    
dotnet_install: Warning: Unable to locate libunwind. Probable prerequisite missing; install libunwind.                
dotnet-install: Downloading link: https://dotnetcli.azureedge.net/dotnet/Sdk/5.0.100-alpha1-013788/dotnet-sdk-5.0.100-alpha1-013788-linux-arm64.tar.gz
dotnet-install: Extracting zip from https://dotnetcli.azureedge.net/dotnet/Sdk/5.0.100-alpha1-013788/dotnet-sdk-5.0.100-alpha1-013788-linux-arm64.tar.gz
dotnet-install: Adding to current process PATH: `/home/omajid/AspNetCore/.dotnet`. Note: This change will be visible only when sourcing script.
dotnet-install: Installation finished successfully.
  Restore completed in 214.81 ms for /home/omajid/.nuget/packages/microsoft.dotnet.arcade.sdk/1.0.0-beta.19462.4/tools/Tools.proj.
dotnet_install: Warning: Unable to locate libunwind. Probable prerequisite missing; install libunwind.
dotnet-install: Downloading link: https://dotnetcli.azureedge.net/dotnet/Runtime/5.0.0-alpha1.19470.7/dotnet-runtime-5.0.0-alpha1.19470.7-linux-x64.tar.gz
dotnet-install: Extracting zip from https://dotnetcli.azureedge.net/dotnet/Runtime/5.0.0-alpha1.19470.7/dotnet-runtime-5.0.0-alpha1.19470.7-linux-x64.tar.gz
dotnet-install: Adding to current process PATH: `/home/omajid/AspNetCore/.dotnet`. Note: This change will be visible only when sourcing script.
dotnet-install: Installation finished successfully.
dotnet_install: Warning: Unable to locate libunwind. Probable prerequisite missing; install libunwind.
dotnet-install: .NET Core Runtime version 5.0.0-alpha1.19470.7 is already installed.
dotnet-install: Adding to current process PATH: `/home/omajid/AspNetCore/.dotnet`. Note: This change will be visible only when sourcing script.
dotnet-install: Installation finished successfully.
  Restore completed in 613.33 ms for /home/omajid/AspNetCore/eng/tools/RepoTasks/RepoTasks.csproj.
/home/omajid/.nuget/packages/microsoft.net.compilers.toolset/3.4.0-beta1-19456-03/tasks/netcoreapp2.1/Microsoft.CSharp.Core.targets(59,5): error : Failed to load �#4, error: /home/omajid/AspNetCore/.dotnet/host/fxr/5.0.0-alpha1.19470.7/libhostfxr.so: cannot open shared object file: No such file or directory [/home/omajid/AspNetCore/eng/tools/RepoTasks/
RepoTasks.csproj]
/home/omajid/.nuget/packages/microsoft.net.compilers.toolset/3.4.0-beta1-19456-03/tasks/netcoreapp2.1/Microsoft.CSharp.Core.targets(59,5): error : The library libhostfxr.so was found, but loading it from /home/omajid/AspNetCore/.dotnet/host/fxr/5.0.0-alpha1.19470.7/libhostfxr.so failed [/home/omajid/AspNetCore/eng/tools/RepoTasks/RepoTasks.csproj]
/home/omajid/.nuget/packages/microsoft.net.compilers.toolset/3.4.0-beta1-19456-03/tasks/netcoreapp2.1/Microsoft.CSharp.Core.targets(59,5): error :   - Installing .NET Core prerequisites might help resolve this problem. [/home/omajid/AspNetCore/eng/tools/RepoTasks/RepoTasks.csproj]
/home/omajid/.nuget/packages/microsoft.net.compilers.toolset/3.4.0-beta1-19456-03/tasks/netcoreapp2.1/Microsoft.CSharp.Core.targets(59,5): error :      https://go.microsoft.com/fwlink/?linkid=2063370 [/home/omajid/AspNetCore/eng/tools/RepoTasks/RepoTasks.csproj]
Build failed (exit code '1'). 
```

As I said in the original PR comments, the `arm64` and `x64` runtimes are slightly different (`5.0.0-alpha1.19415.3` vs `5.0.0-alpha1.19470.7`)... or overwritten? The `x64` hostfxr is newer and wins at the end, resulting in anon-executible binary.

---

:ok: this won't work until `dotnet-install.sh` is fixed. @omajid please file a dotnet/cli issue because that seems to be where the problem lies.

Alternative suggestion below.